File: pset6/mario/mario.py
from cs50 import get_int
height = get_int("Height: ")

if(int(height) > 8 or int(height) < 1 ) :
    height = get_int("Height: ")

height = int(height)

for line in range(1 , height+1):
    spaces = height - line
    for space in range(spaces) :
        print(" ",end = "")
    for brick in range(line) :
        print("#", end = "")
    print("  ",end = "")
    for brick in range(line) :
        print("#", end = "")
    print("")

File: pset1/Mario/mario.c
#include <stdio.h>
#include <cs50.h>


int main() {
    int height = 0 ;
    do {
        height = get_int("Height:\n");
    }while(height > 8 || height <= 0);
    for (int i = 1 ; i <= height ; i++) {
        for(int space = 0 ; space < height -i ; space ++ ) {
            printf(" ");
        }
        for (int hash = 0 ; hash < i ; hash++ ) {
            printf("#");
        }
        printf("  ");
        for (int hash = 0 ; hash < i ; hash++ ) {
            printf("#");
        }
        printf("\n");
    }
    return 0;
}